src/Castle.Core/DynamicProxy/IInterceptorSelector.cs

```diff
@@ -26,7 +26,7 @@ public interface IInterceptorSelector
 		/// <summary>
 		///   Selects the interceptors that should intercept calls to the given <paramref name = "method" />.
 		/// </summary>
-		/// <param name = "type">The type declaring the method to intercept.</param>
+		/// <param name = "type">The type of the target class.</param>
```

Comment:
This is in response to https://github.com/castleproject/Core/issues/74#issuecomment-69142139:

> Unfortunately we've also got a mismatch between behaviour and documented behaviour via XML comments. `type` is implemented as target type, and `method` is implemented as documented, you can get the type declaring the method to intercept via `method.DeclaringType`.